app/domain/entities/auth/password.py

```
@@ -0,0 +1,30 @@
+        if isinstance(value, Password):
+            return value
+
+        try:
+            settings.fernet.decrypt(value.encode())
+            encrypted = value
+        except InvalidToken:
+            encrypted = settings.fernet.encrypt(value.encode()).decode()
+        return cls(encrypted)
+
+    @classmethod
+    def __get_pydantic_json_schema__(cls, core_schema, handler) -> None:
```

Ruff (0.12.2)

Missing type annotation for function argument `core_schema`

(ANN001)